ebml_matroska.xml

```diff
@@ -419,19 +419,23 @@
     <documentation lang="en">Settings describing the colour format.</documentation>
   </element>
   <element name="MatrixCoefficients" path="0*1(\Segment\Tracks\TrackEntry\Video\Colour\MatrixCoefficients)" cppname="VideoColourMatrix" id="0x55B1" type="uinteger" maxOccurs="1" default="2" minver="4" webm="0">
-    <documentation lang="en">The Matrix Coefficients of the video used to derive luma and chroma values from red, green, and blue color primaries. For clarity, the value and meanings for MatrixCoefficients are adopted from Table 4 of ISO/IEC 23001-8:2013/DCOR1.</documentation>
+    <documentation lang="en">The Matrix Coefficients of the video used to derive luma and chroma values from red, green, and blue color primaries. For clarity, the value and meanings for MatrixCoefficients are adopted from Table 4 of ISO/IEC 23001-8:2013/DCOR1 or ITU-T H.273.</documentation>
     <restriction>
       <enum value="0" label="GBR"/>
-      <enum value="1" label="BT709"/>
+      <enum value="1" label="ITU-R BT.709"/>
       <enum value="2" label="unspecified"/>
       <enum value="3" label="reserved"/>
       <enum value="4" label="FCC"/>
-      <enum value="5" label="BT470BG"/>
+      <enum value="5" label="ITU-R BT.470BG"/>
       <enum value="6" label="SMPTE 170M"/>
       <enum value="7" label="SMPTE 240M"/>
       <enum value="8" label="YCoCg"/>
       <enum value="9" label="BT2020 Non-constant Luminance"/>
       <enum value="10" label="BT2020 Constant Luminance"/>
+      <enum value="11" label="SMPTE 2085"/>
```

Comment:
SMPTE ST 2085 (SMPTE uses now "ST" for standards; see e.g. H.265 spec, it uses "ST" for this label).
Generally speaking, if you change old labels I suggest to fix all issues in one shot:
0 = Identity (may be used for XYZ too, which is not GBR)
4 = FCC 73.682
9&10 = ITU-R BT.2020 ...

and to have 2 commits (1 for fixing old labels and 1 for adding new labels)